front-end/src/components/common/Loader/Loader.module.scss

```diff
@@ -1,4 +1,37 @@
-.loader {
+@import 'styles/colors.scss';
+
+.loaderWhite {
```

Comment:
I think you should add two new classes only with bgcolor properties, because you duplicate a lot of properties.

Reply:
I try it, but it didn't work

Reply:
fix it